migrations/alembic.ini

```diff
@@ -0,0 +1,45 @@
+# A generic, single database configuration.
```

Comment:
There's a copy of the `alembic.ini` file at the root of the repo - consolidate this with that